ansible/roles/candlepin/tasks/common/cp_mariadb.yml

```diff
@@ -88,6 +91,20 @@
     name: candlepin
     priv: "candlepin.*:ALL,GRANT"
     state: present
+  when:
```

Comment:
If we're stuck doing part of these DB changes differently based on the OS, I think I'd rather leave the whole thing in each directory as it was -- especially if we're going to be staying at only 2 or 3 different machines.

Reply:
Done.